docs/formcontrols.html

```diff
@@ -3,14 +3,19 @@
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <link href="style.css" rel="stylesheet">
+  <link href="../dist/add-style.min.css" rel="stylesheet">
```

Comment:
I believe this `style.css` came from the NPM script `build-docs` in `package.json`, if it's not used anymore let's clean it up or consistently use the dist or docs style

Reply:
Yeah, this needs some more work. I want to separate the doc styles from add-style and link to the built css like or production apps.